hpctestlib/system/ssh/ssh_audit.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class ssh_audit_check(rfm.RunOnlyRegressionTest):
+    '''ssh audit config test.
+
+    `ssh-audit is a tool for ssh server & client configuration auditing.
+
+    The check consist on performing the basic ssh server config auditing
+    using the master version of https://github.com/jtesta/ssh-audit.
+    '''
+
+    executable = './ssh-audit.py'
+    executable_opts = ['-n', '-l', 'fail', 'localhost']
+    sourcesdir = 'https://github.com/jtesta/ssh-audit'
+    tags = {'system', 'ssh'}
```

Comment:
Should we add a security-related tag? Maybe just `security` or something more specific.

Reply:
I don't know. I don't want to give the impression that this is a real security check. what do you think?